### PR TITLE
plugins/rustaceanvim: fix deprecation in test

### PR DESCRIPTION
**Pull request by khaneliman**

### CODE REVIEW

File: tests/test-sources/plugins/languages/rust/rustaceanvim.nix [hunk 1]
```diff
@@ -115,7 +115,7 @@
               return { 'rust-analyzer', '--log-file', RustaceanConfig.server.logfile }
             end
           '';
-          settings = ''
```
Comment (MattSturgeon) [inline tests/test-sources/plugins/languages/rust/rustaceanvim.nix:118]: Should this be `default_settings.rust-analyzer` as it's a function?

Comment (khaneliman) [reply tests/test-sources/plugins/languages/rust/rustaceanvim.nix:118]: No, the attrset is under rust-analyzer

Comment (MattSturgeon) [reply tests/test-sources/plugins/languages/rust/rustaceanvim.nix:118]: You're right, I read the note in the warning backwards 😅

**Review by MattSturgeon — APPROVED** [review]
LGTM, thanks again!

**Review by traxys — APPROVED** [review]
My bad

**Review by traxys — COMMENTED** [review]
This should be default_settings.rust-analyzer, not that it changes anything though

**Comment by khaneliman** [comment]
> This should be default_settings.rust-analyzer, not that it changes anything though

Our warning says that the function declaration goes under default_settings and attribute sets go under that

**Review by traxys — COMMENTED** [review]
Indeed, sorry misread the example x)

**Comment by MattSturgeon** [comment]
@mergifyio queue

**Pull request merged**